### PR TITLE
Release v0.0.1

### PR DESCRIPTION
**Pull request by Hoverbear**
Following CONTRIBUTING.md

### CODE REVIEW

File: upload_s3.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ done
 for file in $(find "$GITHUB_SHA" -type f); do
   artifact_path="$DEST"/"$(basename "$artifact")"
   md5="$(md5sum "$artifact" | cut -d' ' -f1)"
-  obj="$(aws s3api head-object --bucket "$AWS_BUCKET" --key "$artifact_path")"
+  obj="$(aws s3api head-object --bucket "$AWS_BUCKET" --key "$artifact_path" || echo '{}')"
```
Comment (cole-h) [inline upload_s3.sh:27]: (I suggested this.) A liiiiiiiiittle bit hacky, but I think it'll be fine.

**Review by cole-h — APPROVED** [review]
Woot!

**Comment by Hoverbear** [comment]
We can probably remove those github actions since buildkite does it,....

**Review by cole-h — APPROVED** [review]
Now LGTM, after fixing my derps.... :laughing:

**Pull request merged**